mathpackage.py

```diff
@@ -3,16 +3,16 @@
 """
 
 # Definition of radius
-r = 0.43
```

Comment:
Try to give variable  a name that make readers understand what is it. 
Name Convention 
--https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html